grocy/rootfs/var/www/grocy/middleware/HomeAssistantAuthMiddleware.php

```
@@ -0,0 +1,97 @@
+			]);
+
+			$ch = curl_init('http://supervisor/auth');
+			curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'POST');
+			curl_setopt($ch, CURLOPT_POSTFIELDS, $payload);
+			curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+			curl_setopt($ch, CURLOPT_HTTPHEADER, [
+				'Content-Type: application/json',
+				'X-Supervisor-Token: ' . $supervisorToken
+			]);
+
+			$response = curl_exec($ch);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)